dist/common/scripts/scylla-housekeeping

```diff
@@ -138,12 +138,15 @@ def check_version(ar):
             params = params + "&rtype=" + repo_type
         versions = get_json_from_url(version_url + params)
         latest_version = versions["version"]
-        latest_patch_version = versions["latest_patch_version"]
+        latest_patch_version = versions.get("latest_patch_version")
```

Comment:
Why 140 is ok, but 141 isn't?

Reply:
When we use RC, only returned version, without 'latest_path_version'